### ISSUE TITLE
Sort account search results by recent posts

### ISSUE THREAD

**Issue by FediVideos** [mastodon/mastodon#32882]
### Pitch

It would be useful if people searching for accounts by keyword were shown results in order of recent activity. At the moment activity doesn't seem to affect placing in the search results at all.

### Motivation

At the moment search results sometimes show inactive accounts above active ones. This may lead people to think all the accounts are inactive and they don't bother clicking on the other results to check.

In some cases even completely blank accounts are showing up in search results above non-blank ones. This can be very off-putting for people trying to find people to follow.

**Comment by idotj** [comment]
I think it's a good proposal, I just hope that spam/bot accounts don't take advantage of this, as they would surely be the first to position themselves.

**Comment by FediVideos** [comment]
Yeah, someone off Github just said something similar that they don't want clout chasers to be spamming posts just to get to the top of results.

I'll clarify the original post.
